src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -447,19 +438,43 @@ public string NameClaimType
         /// Gets or sets a value indicating whether SAML tokens must have at least one AudienceRestriction.
         /// </summary>
         [DefaultValue(true)]
-        public bool RequireAudience { get; set; }
+        public bool RequireAudience
+        {
+            get
+            {
+                return _requireAudience;
```

Comment:
I might be missing something, why this getter is different for the rest that also have respective private variables? what problem are we solving with private variables?

Reply:
It was extra confusing because I didn't finish doing it for the rest of the properties so the code was incorrect 😳. The private variables are needed here as the constructors for TVP were using the (now obsolete) setters which were causing warnings (which are treated as errors) at compile time.

Reply:
Makes sense, Thanks.